pypowsybl/report.py

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2022, RTE (http://www.rte-france.com)
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+from pypowsybl._pypowsybl import create_reporter_model, print_report, JavaHandle
```

Comment:
we need to take care about namespace pollution:
those objects should not be visible for the user in `pypowsybl.report` module.

In order to achieve it several possibilities:
- use a qualified version in the code, maybe aliasing the module as `_pp` to make it readable, sounds reasonable
- import the names with an added underscore
- or as targetted in future versions, move the code to an inner "implementation" module, and only import here what we want to expose

Reply:
Done (aliased the referred module as _pp)